app/assets/stylesheets/internal/layout.scss

```diff
@@ -1,4 +1,5 @@
 @import '../variables';
+@import '../crayons';
```

Comment:
I was toying with the ideal of just importing _certain_ crayons styles into our `/internal` pages, like colors + banners...but then realized that someone might want to use _other_ crayons variables inside the html, too. This _might_ be too heavyweight to import, but did it here and figured that someone would tell me if this was a bad idea or not :)